docs/source/tutorials/text.ipynb

```diff
@@ -24,6 +24,54 @@
     "- Train a more robust version of the same neural net via cleanlab's `CleanLearning` wrapper.\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "Do you have an sklearn compatible `model`, text `data` and true `labels`? Run the code below to train your `model` and get label issues.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from cleanlab.classification import CleanLearning\n",
+    "\n",
+    "cl = CleanLearning(model)\n",
+    "_ = cl.fit(data, labels)\n",
+    "label_issues = cl.get_label_issues()\n",
```

Comment:
Maybe also add at the end: 

`preds = cl.predict(test_data)  # predictions from a version of your model trained on auto-cleaned data`